server/core/src/main/java/org/infinispan/server/core/AbstractProtocolServer.java

```diff
@@ -155,6 +139,9 @@ public String getHost() {
    }
 
    public int getPort() {
+      if(transport != null) {
```

Comment:
Yeah I do.... but I've never done spaces between `if` and `brackets` :) My fingers are faster then IDE :)